docs/configuration.md

```diff
@@ -229,6 +243,30 @@ log:
     </td>
 </table>
 
+## hooks
+
+
+```yaml 
+hooks:
+  - type: mail
+    levels:
+      - panic
+    options:
+      smtp:
+        addr: smtp.sendhost.com:25
+        username: sendername
+        password: password
+        insecure: true
+      from: name@sendhost.com
+      to:
+        - name@receivehost.com
+```
+
+The `hooks` subsection configures the logging hooks' behavior, including a 
```

Comment:
Thank you for the contribution. A few tweaks to improve the readability. Also, could you remove the TODO, we don't leave those in our documentation --- where you see them is really where we are exposing specifications for convenience.

--> https://gist.github.com/moxiegirl/4c9f4e110807a559d201
The `hooks` subsection configures the logging hooks' behavior. This subsection includes a sequence handler which you can use for sending mail, for example. Refer to `loglevel` to configure the level of messages printed.